.github/workflows/github_actions.yml

```diff
@@ -45,7 +45,7 @@ jobs:
     runs-on: ${{ matrix.os }}
     strategy:
       matrix:
-        os: [ ubuntu-latest, macos-11 ]
+        os: [ ubuntu-latest, macos-12 ]
```

Comment:
Use `macos-latest` like Ubuntu?

Reply:
```macos-latest``` is a dynamic image, so i am trying to avoid updates that could lead to breaking changes. With this in mind we should consider using ```ubuntu-22.04``` instead of ```ubuntu-latest```